src/Exception/ExceptionInterface.php

```diff
@@ -1,7 +1,11 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Webimpress\SafeWriter\Exception;
 
-interface ExceptionInterface
+use Throwable;
+
+interface ExceptionInterface extends Throwable
```

Comment:
Suffixes like `Interface` are discouraged by the Doctrine coding standard since they don't carry any meaning from the API standpoint. Would it make sense to rename it to `Webimpress\SafeWriter\Exception`?

Reply:
I am using `webimpress/coding-standard` here and it is one of the rules to have `Interface` suffix. The same approach we have in zendframework, and it's also my personal preference. 

Recently in work we had issue with service called `User` and we had also entity called `User`. In many places it caused conflict and context was unclear. In most places where we used service we import it with alias `UserService` and finally we renamed it, so now all is clear.